pets/cassandra/on-start.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+# Parse out hostname, formatted as: petset_name-index
+IFS='-' read -ra ADDR <<< "$(hostname)"
+MY_ID=$(expr "1" + "${ADDR[1]}")
+CFG=/etc/cassandra/cassandra.yaml
+
+i=0
+while read -ra LINE; do
+    let i=i+1
+    if [ $i = $MY_ID ]; then
+        MY_NAME=$LINE
+    fi
+    PEERS=("${PEERS[@]}" $LINE)
+done
```

Comment:
So this bash is hurting my head.  I am have enough humility to admit it. So how the hell is this setting the seeds?